tutorials/cloud-run-multiregional-deployments/code/lb.tf

```
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+resource "google_compute_region_network_endpoint_group" "cloud_run_neg" {
+  for_each = google_cloud_run_service.default
+
+  name                  = "cloud-run-neg"
+  network_endpoint_type = "SERVERLESS"
+  region                = each.value.location
+  project               = data.google_project.project.project_id
+  cloud_run {
+    service = each.value.name
+  }
+}
+
+
+module "lb-http" {
+  source  = "GoogleCloudPlatform/lb-http/google//modules/serverless_negs"
+  version = "~> 5.1"
```

```suggestion
  version = "~> 6.3"
```

Bump to latest version -> works fine.

---

Thanks for the update @kunzese. Hopefully this works for you now.